.devcontainer/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/dotnet/.devcontainer/base.Dockerfile
 
-# [Choice] .NET version: 5.0, 3.1, 2.1
-ARG VARIANT="5.0"
-FROM mcr.microsoft.com/vscode/devcontainers/dotnet
+# full tag list: https://mcr.microsoft.com/v2/devcontainers/dotnet/tags/list
+ARG VARIANT="8.0"
+FROM mcr.microsoft.com/devcontainers/dotnet:${VARIANT}
```

Comment:
Can we use `latest` here so we avoid having to maintain the VARIANT in this file?

We end up installing our own .NET version anyways for the repo.

Reply:
ok, keep the argument in case of we want use a specific tag for whatever  reason, if it's no need then I would remove the argument both from Dockerfile and devcontainer.json, looks good?

Reply:
> if it's no need then I would remove the argument both from Dockerfile and devcontainer.json, looks good?

Yep, removing them from both sounds good.